.github/workflows/generate_status_page.yaml

```diff
@@ -0,0 +1,22 @@
+name: Generate status page
+
+on: workflow_dispatch
+
+jobs:
+  generate-status-page:
+    concurrency:
+      group: generate-status-page
+      cancel-in-progress: true
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+      fail-fast: true
```

Comment:
Seems like the matrix config is redundant. This is a single job, isn't it?

Reply:
it is, but I normally use OS, library or tool version variables on matrix, no matter whether this is single or not in case the job needs to be refactored to cover different versions or OSs